geode-core/src/main/java/org/apache/geode/internal/cache/DistributedCacheOperation.java

```diff
@@ -345,7 +345,8 @@ protected void _distribute() {
 
     try {
       // Recipients with CacheOp
-      Set<InternalDistributedMember> recipients = getRecipients();
+      Set<InternalDistributedMember> recipients = new HashSet<>();
```

Comment:
Why are you copying the recipient set?
Also, if copying is desired use `new HashSet<>(getRecipients())`, which results in a properly sized `HashSet` on allocation.

Reply:
getRecipients() returns unmodified Set, and code (in _distribute method) few lines after, isperforming removeAll on returned Set.